bitmovin/resources/models/encodings/pertitle/per_title.py

```diff
@@ -0,0 +1,33 @@
+from bitmovin.errors import InvalidTypeError
+from bitmovin.utils import Serializable
+from .h264_per_title_configuration import H264PerTitleConfiguration
+
+
+class PerTitle(Serializable):
+    def __init__(self, h264_configuration=None):
+        super().__init__()
+
+        self._h264_configuration = None
+        self.h264_configuration = h264_configuration
+
+    @property
+    def h264_configuration(self):
+        return self._h264_configuration
+
+    @h264_configuration.setter
+    def h264_configuration(self, new_h264_configuration):
+        if new_h264_configuration is None:
+            self._h264_configuration = None
```

Comment:
you can return here?

Reply:
You have to return here :) otherwise this won't work